app/components/header/view.html.erb

```diff
@@ -22,6 +22,11 @@
         <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation" data-module="govuk-button">Menu</button>
         <nav>
           <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+            <% if @current_user.system_admin? %>
```

Comment:
https://github.com/DFE-Digital/register-trainee-teacher-data/blob/af58e74bd50e72cee2aed8a4181e285d2499343e/app/components/header/view.rb#L13-L15

Can we take that approach instead please